t/venom/scenarios/template/playbooks/run_tests.yml

```diff
@@ -1,6 +1,6 @@
 ---
 - hosts: pfservers
-  name: Run unit tests
+  name: Run Venom tests
   become: True
 
   vars:
```

Comment:
Shouldn't we had here the part of the "Check configurator state" as a template?

Reply:
If you agree, let's keep it like this for now. We will have to rework how Ansible run tests for each scenario to avoid duplication of code.